manifests/base/webhook/manifests.yaml

```diff
@@ -0,0 +1,26 @@
+---
+apiVersion: admissionregistration.k8s.io/v1
+kind: ValidatingWebhookConfiguration
+metadata:
+  name: validating-webhook-configuration
+webhooks:
+- admissionReviewVersions:
+  - v1
+  clientConfig:
+    service:
+      name: webhook-service
+      namespace: system
+      path: /validate-kubeflow-org-v1-pytorchjob
```

Comment:
We need to follow this form based on the https://book.kubebuilder.io/reference/markers/webhook.
Also, since this path is global, we need to make an effort to avoid conflicts.